ci/docker/build/Dockerfile

```diff
@@ -59,10 +59,8 @@ RUN apt-get install -y \
     mtools             \
     nasm               \
     net-tools          \
-    openjdk-8-jdk      \
```

Comment:
Why are these being removed? Are they no longer needed, or do they appear in the new image by default?

Reply:
They do not exist on the default packages anymore and seem like they are not needed. Waiting for the cirrus test to complete to validate they are not needed anymore.

Reply:
Confirmed that they are not needed anymore.